.env.production

```
@@ -1,11 +1,11 @@
 # Supabase Configuration - Production
-EXPO_PUBLIC_SUPABASE_URL=https://xagpimsaqrwfpsduqzea.supabase.co
-EXPO_PUBLIC_SUPABASE_ANON_KEY=sb_publishable_R2GV4BSjcrrzM1VICU13Aw_c0O2yh-A
+EXPO_PUBLIC_SUPABASE_URL=https://pdpqkgrqlubyzkcivifk.supabase.co
+EXPO_PUBLIC_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZSIsInJlZiI6InBkcHFrZ3JxbHVieXprY2l2aWZrIiwicm9sZSI6ImFub24iLCJpYXQiOjE3NTkyMDk3MzgsImV4cCI6MjA3NDc4NTczOH0.xcjdhR89okeqaMGeq5oYAjuvRk56H9-Wc8SXjHpPls4
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The EXPO_PUBLIC_SUPABASE_ANON_KEY key should go before the EXPO_PUBLIC_SUPABASE_URL key

(UnorderedKey)